app/views/content/index.html.erb

```diff
@@ -1,5 +1,18 @@
 <% content_for :title, @presenter.title %>
 <% content_for :page_class, "content-data__index" %>
+<%
+  if params[:submitted]
```

Comment:
Can we move these out of the view?

Reply:
I tried but couldn't access the "submitted" param from the presenter - happy to fix if shown how.

Reply:
You'd have to pass the params to the presenter, but I'm not sure if that's a good idea myself.

Reply:
I've updated this - I no longer make the blank selectable at the top conditional, and I pass the presenter the page params and work with it to figure out whether to select the blank option or the explicit "All" option.